dbt/adapters/spark/column.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+from typing import TypeVar
+
+from dbt.adapters.base.column import Column
+
+Self = TypeVar('Self', bound='SparkColumn')
+
+
+@dataclass(init=False)
+class SparkColumn(Column):
+    column: str
+    dtype: str
+    comment: str
+
+    def __init__(
+        self,
+        column: str,
+        dtype: str,
+        comment: str = None
```

Comment:
If comments can be None, instead just do:
```
@dataclass
class SparkColumn(Column):
  column: str
  dtype: str
  comment: Optional[str]
```

That will add `comment` as a new optional str field, which is what it is based on this argument spec.

Reply:
Perfect, thanks.